oneshot: false

reverse_tf: false

localization_freq: 100

# for daikin plant model, [(1.5, 1.2, 0), (0.0, 0.0, 0.33)] is around the ground truth
initial_guess:
        pos: [0, 0, 0] # OK: (0.0, 0.0, 0.0), (0.0, 1.0, 0), (1.0, 1.0, 0), (1.5, 1.0, 0)
        rpy: [0, 0, 0] # yaw: (xy: 1.5, 1.2): 0, 0.6, 0.8;  (xy: 0, 0): 0, 0.8

registration:
        scan_stack_size: 10
        init:
                map_voxel_size: 0.2 # 0.2: best
                scan_voxel_size: 0.2 # 0.2: best
                max_corres_dist: 0.4 # 0.4: best
                localization_thresh: 0.8
                map_range: 10.0
                scan_range: 10.0 
        float:
                map_voxel_size: 0.1
                scan_voxel_size: 0.1
                max_corres_dist: 0.2
                localization_thresh: 0.8
                map_range: 5.0 
                scan_range: 5.0
        fix:
                map_voxel_size: 0.04 # 0.02
                scan_voxel_size: 0.04 # 0.02
                max_corres_dist: 0.08 # 0.02
                localization_thresh: 0.9
                map_range: 3.0
                scan_range: 3.0
